README.md
iso-snapshot is a tool used for creating a live-CD from the running system.


debian/changelog
iso-snapshot-antix (0.2.2) unstable; urgency=medium

  * exclude localhost only for generalized snapshot 

 -- anticapitalista <antix@operamail.com>  Thu, 02 Nov 2017 00:20:44 +0200

iso-snapshot-antix (0.2.1) unstable; urgency=medium

  * Various cosmetic changes made upstream in mx-snapshot
  * Lithuanian and Slovak added
  
 -- anticapitalista <antix@operamail.com>  Wed, 25 Oct 2017 14:37:47 +0300

iso-snapshot-antix (0.2.0) unstable; urgency=medium

  * bump version
  * add persist-config.conf to excludes list

 -- anticapitalista <antix@operamail.com>  Fri, 13 Oct 2017 12:43:35 +0300

iso-snapshot-antix (0.1.12) unstable; urgency=medium

  * Use capital letters in desktop file

 -- anticapitalista <antix@operamail.com>  Sat, 07 Oct 2017 21:04:17 +0300

iso-snapshot-antix (0.1.11) unstable; urgency=medium

  * New copy-initrd-modules script

 -- anticapitalista <antix@operamail.com>  Sat, 15 Jul 2017 12:03:42 +0100

iso-snapshot-antix (0.1.10) unstable; urgency=medium

  * Added exclusion for network connections

 -- anticapitalista <antix@operamail.com>  Sat, 24 Jun 2017 12:16:23 +0300

iso-snapshot-antix (0.1.9) unstable; urgency=medium

  * New copy-initrd-modules for encrytied live usb

 -- anticapitalista <antix@operamail.com>  Sat, 22 Apr 2017 02:45:29 +0300

iso-snapshot-antix (0.1.8) unstable; urgency=medium

  * Remove virtualbox-guest depends

 -- anticapitalista <antix@operamail.com>  Mon, 10 Apr 2017 20:48:39 +0300

iso-snapshot-antix (0.1.7) unstable; urgency=low

  * Added Czech and Portuguese translations

 -- anticapitalista <antix@operamail.com>  Sun, 02 Apr 2017 19:11:49 +0300
 
iso-snapshot-antix (0.1.6) unstable; urgency=medium

  * Many changes, clean ups to sync with upstream

 -- anticapitalista <antix@operamail.com>  Tue, 21 Feb 2017 10:57:49 +0200

iso-snapshot-antix (0.1.5) unstable; urgency=medium

  * apt-get -y install used

 -- anticapitalista <antix@operamail.com>  Sun, 05 Feb 2017 15:39:06 +0200

iso-snapshot-antix (0.1.4) unstable; urgency=medium

  * Updated copy-initrd-modules file

 -- anticapitalista <antix@operamail.com>  Sun, 08 Jan 2017 23:21:01 +0200

iso-snapshot-antix (0.1.3) unstable; urgency=medium

  * newer copy-to-init

 -- anticapitalista <antiX@operamail.com>  Wed, 06 Apr 2016 11:29:36 -0400

iso-snapshot-antix (0.1.2) unstable; urgency=medium

  * bigfixes 

 -- anticapitalista <antiX@operamail.com>  Mon, 21 Mar 2016 07:05:09 -0400

iso-snapshot-antix (0.1.1) unstable; urgency=low 
 
  * Added translations.

 -- anticapitalista <antiX@operamail.com>  Thu, 31 Dec 2015 19:52:00 +0200


iso-snapshot-antix (0.1) unstable; urgency=low 
 
  * Initial Release.

 -- anticapitalista <antiX@operamail.com>  Mon, 28 Dec 2015 14:58:00 +0200


moc_isosnapshot.cpp
/****************************************************************************
** Meta object code from reading C++ file 'isosnapshot.h'
**
** Created by: The Qt Meta Object Compiler version 67 (Qt 5.7.1)
**
** WARNING! All changes made in this file will be lost!
*****************************************************************************/

#include "isosnapshot.h"
#include <QtCore/qbytearray.h>
#include <QtCore/qmetatype.h>
#if !defined(Q_MOC_OUTPUT_REVISION)
#error "The header file 'isosnapshot.h' doesn't include <QObject>."
#elif Q_MOC_OUTPUT_REVISION != 67
#error "This file was generated using the moc from 5.7.1. It"
#error "cannot be used with the include files from this version of Qt."
#error "(The moc has changed too much.)"
#endif

QT_BEGIN_MOC_NAMESPACE
struct qt_meta_stringdata_isosnapshot_t {
    QByteArrayData data[25];
    char stringdata0[504];
};
#define QT_MOC_LITERAL(idx, ofs, len) \
    Q_STATIC_BYTE_ARRAY_DATA_HEADER_INITIALIZER_WITH_OFFSET(len, \
    qptrdiff(offsetof(qt_meta_stringdata_isosnapshot_t, stringdata0) + ofs \
        - idx * sizeof(QByteArrayData)) \
    )
static const qt_meta_stringdata_isosnapshot_t qt_meta_stringdata_isosnapshot = {
    {
QT_MOC_LITERAL(0, 0, 11), // "isosnapshot"
QT_MOC_LITERAL(1, 12, 9), // "procStart"
QT_MOC_LITERAL(2, 22, 0), // ""
QT_MOC_LITERAL(3, 23, 8), // "procTime"
QT_MOC_LITERAL(4, 32, 8), // "procDone"
QT_MOC_LITERAL(5, 41, 14), // "setConnections"
QT_MOC_LITERAL(6, 56, 13), // "disconnectAll"
QT_MOC_LITERAL(7, 70, 17), // "onStdoutAvailable"
QT_MOC_LITERAL(8, 88, 22), // "on_buttonAbout_clicked"
QT_MOC_LITERAL(9, 111, 21), // "on_buttonBack_clicked"
QT_MOC_LITERAL(10, 133, 23), // "on_buttonCancel_clicked"
QT_MOC_LITERAL(11, 157, 27), // "on_buttonEditConfig_clicked"
QT_MOC_LITERAL(12, 185, 28), // "on_buttonEditExclude_clicked"
QT_MOC_LITERAL(13, 214, 21), // "on_buttonHelp_clicked"
QT_MOC_LITERAL(14, 236, 21), // "on_buttonNext_clicked"
QT_MOC_LITERAL(15, 258, 31), // "on_buttonSelectSnapshot_clicked"
QT_MOC_LITERAL(16, 290, 27), // "on_excludeDocuments_toggled"
QT_MOC_LITERAL(17, 318, 7), // "checked"
QT_MOC_LITERAL(18, 326, 27), // "on_excludeDownloads_toggled"
QT_MOC_LITERAL(19, 354, 26), // "on_excludePictures_toggled"
QT_MOC_LITERAL(20, 381, 23), // "on_excludeMusic_toggled"
QT_MOC_LITERAL(21, 405, 24), // "on_excludeVideos_toggled"
QT_MOC_LITERAL(22, 430, 25), // "on_excludeDesktop_toggled"
QT_MOC_LITERAL(23, 456, 22), // "on_radioRespin_clicked"
QT_MOC_LITERAL(24, 479, 24) // "on_radioPersonal_clicked"

    },
    "isosnapshot\0procStart\0\0procTime\0"
    "procDone\0setConnections\0disconnectAll\0"
    "onStdoutAvailable\0on_buttonAbout_clicked\0"
    "on_buttonBack_clicked\0on_buttonCancel_clicked\0"
    "on_buttonEditConfig_clicked\0"
    "on_buttonEditExclude_clicked\0"
    "on_buttonHelp_clicked\0on_buttonNext_clicked\0"
    "on_buttonSelectSnapshot_clicked\0"
    "on_excludeDocuments_toggled\0checked\0"
    "on_excludeDownloads_toggled\0"
    "on_excludePictures_toggled\0"
    "on_excludeMusic_toggled\0"
    "on_excludeVideos_toggled\0"
    "on_excludeDesktop_toggled\0"
    "on_radioRespin_clicked\0on_radioPersonal_clicked"
};
#undef QT_MOC_LITERAL

static const uint qt_meta_data_isosnapshot[] = {

 // content:
       7,       // revision
       0,       // classname
       0,    0, // classinfo
      22,   14, // methods
       0,    0, // properties
       0,    0, // enums/sets
       0,    0, // constructors
       0,       // flags
       0,       // signalCount

 // slots: name, argc, parameters, tag, flags
       1,    0,  124,    2, 0x0a /* Public */,
       3,    0,  125,    2, 0x0a /* Public */,
       4,    1,  126,    2, 0x0a /* Public */,
       5,    0,  129,    2, 0x0a /* Public */,
       6,    0,  130,    2, 0x0a /* Public */,
       7,    0,  131,    2, 0x0a /* Public */,
       8,    0,  132,    2, 0x08 /* Private */,
       9,    0,  133,    2, 0x08 /* Private */,
      10,    0,  134,    2, 0x08 /* Private */,
      11,    0,  135,    2, 0x08 /* Private */,
      12,    0,  136,    2, 0x08 /* Private */,
      13,    0,  137,    2, 0x08 /* Private */,
      14,    0,  138,    2, 0x08 /* Private */,
      15,    0,  139,    2, 0x08 /* Private */,
      16,    1,  140,    2, 0x08 /* Private */,
      18,    1,  143,    2, 0x08 /* Private */,
      19,    1,  146,    2, 0x08 /* Private */,
      20,    1,  149,    2, 0x08 /* Private */,
      21,    1,  152,    2, 0x08 /* Private */,
      22,    1,  155,    2, 0x08 /* Private */,
      23,    1,  158,    2, 0x08 /* Private */,
      24,    1,  161,    2, 0x08 /* Private */,

 // slots: parameters
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void, QMetaType::Int,    2,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void,
    QMetaType::Void, QMetaType::Bool,   17,
    QMetaType::Void, QMetaType::Bool,   17,
    QMetaType::Void, QMetaType::Bool,   17,
    QMetaType::Void, QMetaType::Bool,   17,
    QMetaType::Void, QMetaType::Bool,   17,
    QMetaType::Void, QMetaType::Bool,   17,
    QMetaType::Void, QMetaType::Bool,   17,
    QMetaType::Void, QMetaType::Bool,   17,

       0        // eod
};

void isosnapshot::qt_static_metacall(QObject *_o, QMetaObject::Call _c, int _id, void **_a)
{
    if (_c == QMetaObject::InvokeMetaMethod) {
        isosnapshot *_t = static_cast<isosnapshot *>(_o);
        Q_UNUSED(_t)
        switch (_id) {
        case 0: _t->procStart(); break;
        case 1: _t->procTime(); break;
        case 2: _t->procDone((*reinterpret_cast< int(*)>(_a[1]))); break;
        case 3: _t->setConnections(); break;
        case 4: _t->disconnectAll(); break;
        case 5: _t->onStdoutAvailable(); break;
        case 6: _t->on_buttonAbout_clicked(); break;
        case 7: _t->on_buttonBack_clicked(); break;
        case 8: _t->on_buttonCancel_clicked(); break;
        case 9: _t->on_buttonEditConfig_clicked(); break;
        case 10: _t->on_buttonEditExclude_clicked(); break;
        case 11: _t->on_buttonHelp_clicked(); break;
        case 12: _t->on_buttonNext_clicked(); break;
        case 13: _t->on_buttonSelectSnapshot_clicked(); break;
        case 14: _t->on_excludeDocuments_toggled((*reinterpret_cast< bool(*)>(_a[1]))); break;
        case 15: _t->on_excludeDownloads_toggled((*reinterpret_cast< bool(*)>(_a[1]))); break;
        case 16: _t->on_excludePictures_toggled((*reinterpret_cast< bool(*)>(_a[1]))); break;
        case 17: _t->on_excludeMusic_toggled((*reinterpret_cast< bool(*)>(_a[1]))); break;
        case 18: _t->on_excludeVideos_toggled((*reinterpret_cast< bool(*)>(_a[1]))); break;
        case 19: _t->on_excludeDesktop_toggled((*reinterpret_cast< bool(*)>(_a[1]))); break;
        case 20: _t->on_radioRespin_clicked((*reinterpret_cast< bool(*)>(_a[1]))); break;
        case 21: _t->on_radioPersonal_clicked((*reinterpret_cast< bool(*)>(_a[1]))); break;
        default: ;
        }
    }
}

const QMetaObject isosnapshot::staticMetaObject = {
    { &QDialog::staticMetaObject, qt_meta_stringdata_isosnapshot.data,
      qt_meta_data_isosnapshot,  qt_static_metacall, Q_NULLPTR, Q_NULLPTR}
};


const QMetaObject *isosnapshot::metaObject() const
{
    return QObject::d_ptr->metaObject ? QObject::d_ptr->dynamicMetaObject() : &staticMetaObject;
}

void *isosnapshot::qt_metacast(const char *_clname)
{
    if (!_clname) return Q_NULLPTR;
    if (!strcmp(_clname, qt_meta_stringdata_isosnapshot.stringdata0))
        return static_cast<void*>(const_cast< isosnapshot*>(this));
    return QDialog::qt_metacast(_clname);
}

int isosnapshot::qt_metacall(QMetaObject::Call _c, int _id, void **_a)
{
    _id = QDialog::qt_metacall(_c, _id, _a);
    if (_id < 0)
        return _id;
    if (_c == QMetaObject::InvokeMetaMethod) {
        if (_id < 22)
            qt_static_metacall(this, _c, _id, _a);
        _id -= 22;
    } else if (_c == QMetaObject::RegisterMethodArgumentMetaType) {
        if (_id < 22)
            *reinterpret_cast<int*>(_a[0]) = -1;
        _id -= 22;
    }
    return _id;
}
QT_END_MOC_NAMESPACE
